src/types/video.ts

```diff
@@ -349,4 +359,5 @@ export interface ReactVideoProps extends ReactVideoEvents, ViewProps {
   debug?: DebugConfig;
   allowsExternalPlayback?: boolean; // iOS
   controlsStyles?: ControlsStyles; // Android
+  contextualActions?: ContextualActions[];
```

Comment:
I have a question, can contextualActions be changed during the playback ? If not (and I don't think so), can you please move it inside source property ?
My point about tranlations should also be fixed here, I think you should add a `description` filed into ContextualActions.
And I also wondering if: endAt is the good api ? what will happen if endAt < startAt. A duration would be easier to use ? (This is minor and not blocking the merge)